### ISSUE TITLE
SIF Permissions for all TDS Users 

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#9715]
### SIF Permissions for multiple accounts. All SIF permissions should match MITCHLLA

- [x] CALLISON
- [x] MAJDDARI
- [x] GOLDENB
- [x] GOODJ
- [x] CARDENAM
- [x] MORROWAD
- [x] JOHNSREN
- [x] KONTSEVOYD
- [x] RAHMANMA
- [x] (Need user name) Joan Minyard
- [x] (Need user name) Sean Conway

----

> What application are you using?

AMANDA

> Describe the problem.

The entire TDS team needs permission to edit SIF folders in AMANDA. Team members have had issues with adding rows in the building info tab, uploading attachments, etc. The following users are affected: Justin Good, Joan Minyard, Matiur Rahman, Bryan Golden, Adrianna Morrow, Dari Majd, Jayesh Dongre, Elena Christopher-Allison, Curtis Beaty, Daniil Kontsevoy

> How soon do you need this?

Soon — This week 

> Is there anything else we should know?

This issue is hindering the team's ability to effectively handle the workload associated with SIF.

> Requested By
Nathan A.

[Attachment](https://api.knack.com/v1/applications/595d00ebd315cc4cb98daff4/download/asset/62cd8e712050c5002030a8cf/microsoftteamsimage2.png) (8.32kb)

Request ID: DTS22-104600

**Comment by TracyLinder** [comment]
I'll ask Nathan for a list of user names.

**Comment by TracyLinder** [comment]
User names have been added to the issue description.

**Comment by TracyLinder** [comment]
@Sreelatha-BlazeKannan a few of these already have some SIF permissions but are getting various errors, so I'm wondering if their permissions match Ambers.

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan @TracyLinder i have sent this over to CTM under TASK **SCTASK0232463**

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder  - Updating Andrea's Email 
"""
The SN ticket is SCTASK0232463. I have updated these accounts in V6 PROD except for CONWAYSE who does not have a V6 PROD account, but they do have a V6 TEST account and I will promote it to PROD overnight. They also didn't have a V7 account so I created it.

Note that some of these accounts in V7 had security groups that were at a higher level than the similar as MITCHLLA which had every one at Basic. I added the Basic to these others and let the other security as is. Let me know if I need to go back in and, in some cases, downgrade to just Basic.
"""
